release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -341,6 +341,26 @@ The Cloud Credential Operator utility (`ccoctl`) now creates secrets that use re
 
 With this release, when you xref:../installing/installing_gcp/uninstalling-cluster-gcp.adoc#cco-ccoctl-deleting-sts-resources_uninstalling-cluster-gcp[delete GCP resources with the Cloud Credential Operator utility], you must specify the directory containing the files for the component `CredentialsRequest` objects.
 
+[discrete]
+[id="ocp-4-12-psa-restricted-enforcement"]
+=== Future restricted enforcement for pod security admission
+
+Currently, pod security violations are shown as warnings and logged in the audit logs, but do not cause the pod to be rejected.
+
+Global restricted enforcement for pod security admission is currently planned for the next minor release of {product-title}. When this restricted enforcement is enabled, pods with pod security violations will be rejected.
+
+To prepare for this upcoming change, ensure that your workloads can run as restricted. Workloads that are not configured according to the enforced security standards defined globally or at the namespace level will be rejected. The `restricted-v2` SCC admits workloads according to the link:https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted[Restricted] Kubernetes definition.
```

Comment:
"ensure that your workloads can run as restricted"
This part is a bit inaccurate. You're allowed to have non-restricted workloads but they need to match the global PSa profile (restricted) or the namespace profile (either whatever you set it or what the autolabelling gets you)

Reply:
Thanks! Updated per slack convo